ontology/ngbo.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: ontology_detail
+id: ngbo
+title: Next Generation Biobanking Ontology
+contact:
+  email: dal.alghamdi92@gmail.com
+  github: dalalghamdi
+  label: Dalia Alghamdi
+  orcid: 0000-0002-2801-0767
+description: 'The Next Generation Biobanking Ontology (NGBO) is an open application ontology representing contextual data about omics digital assets in biobank. The ontology focuses on capturing the information about three main activities: wet bench analysis used to generate omics data, bioinformatics analysis used to analyze and interpret data, and data management.'
+domain: investigations
+homepage: https://github.com/Dalalghamdi/NGBO
+issue_requested: 1819
+license:
+  label: CC BY 4.0
+  url: https://creativecommons.org/licenses/by/4.0/
+preferredPrefix: NGBO
+products:
+- id: ngbo.owl
+pull_request_added: 2214
+repository: https://github.com/Dalalghamdi/NGBO
+tracker: https://github.com/Dalalghamdi/NGBO/issues
+activity_status: active
+---
+
```

Comment:
I'd like to see a more detailed description in the free text, do we have a checklist on what to include there?

Reply:
It looks like a lot of context from https://github.com/OBOFoundry/OBOFoundry.github.io/issues/1819 was lost (e.g., Intended use/related projects). that should find its way into the free text

Reply:
@cthoyt Thank you for your time, I edited the textual distribution, hopefully this will resolve the issue. Please let me know I should add more information.